lib/Cli.php

```diff
@@ -451,22 +439,18 @@ protected function convert($vObj)
 
     /**
      * Colorizes a file.
-     *
-     * @param Component $vObj
      */
-    protected function color($vObj)
+    protected function color(Component $vObj): int
     {
-        fwrite($this->stdout, $this->serializeComponent($vObj));
+        $this->serializeComponent($vObj);
```

Comment:
`$this->serializeComponent($vObj)` does its `fwrite` itself and returns `void` so it makes no sense to `fwrite` it itself.